app/templates/editor/article/edit.jade

```diff
@@ -22,12 +22,9 @@ block content
 
   #article-treema
 
-  #article-view
-
   h3(data-i18n="resources.patches") Patches
   .patches-view
 
   hr
 
   div#error-view
```

Comment:
Btw, is this line of code still active? I can't seem to find any reference to #error-view.